docs/site/Decorators.md

```diff
@@ -132,56 +132,181 @@ accordingly or do a composition of them:
+    ) {}
+  }
   ```
 
-  or
+  Writing the whole parameter specification is tedious, we create shortcuts to define params with pattern `@param.${in}.${type}(${name})`:
+
+  - in: the parameter location, one of the following values: `query`, `header`, `path`.
+  - type: a [common name of OpenAPI primitive data type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types), 
+    take specifying a data get from query as an example, a list of available shortcuts can
+    be found in [API document](http://apidocs.loopback.io/@loopback%2fopenapi-v3/#param.query).
+  - name: name of the parameter, should be a `string`, 
 
+  An equivalent example using shortcut decorator would be:
+  
   ```ts
-    // example 2: decorator `@param` applied on parameter level
     class MyController {
-      @get('/')
+      @get('/Pets/{category}')
       list(
-        @param(offsetSpec) offset?: number,
-        @param(pageSizeSpec) pageSize?: number,
+        @param.path.string('category') category: string,
+        @param.query.number('pageSizes') pageSize?: number,
       ) {}
     }
   ```
 
-  In the first example, we apply multiple decorators to a single declaration.
-  The order in which the decorators are called is important because a `@param`
-  decorator must be applied after an operation decorator.
-  To learn more about TypeScript decorator composition,
-  refer to [TypeScript Decorator Documentation](https://www.typescriptlang.org/docs/handbook/decorators.html)
+  You can find the specific usage in [Writing Controller methods](controller.htm#writing-controller-methods)
+
+  *We are supporting parameter location 'cookie' in the feature, track the feature in story*
+  *https://github.com/strongloop/loopback-next/issues/997*
 
-  Please note method level `@param` and parameter level `@param` are mutually exclusive,
-  you can not mix and apply them to the same parameter.
+### RequestBody Decorator
 
-  You can also use this pattern to make it quicker to define params: `@param.${in}.${type}(${name})`
+  Syntax: see [API document](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/request-body-decorator.ts#L20-L79)
 
-  - in: one of the following values: `query`, `header`, `path`, `formData`, `body`
-  - type: one of the following values:  `string`, `number`, `boolean`, `integer`
-  - name: a `string`, name of the parameter
+  `@requestBody()` is applied to a method parameter to generate OpenAPI requestBody specification for it.
+  
+  *Only one parameter can be decorated by `@requestBody` per controller method.*
+
+  A typical OpenAPI requestBody spec contains property `description`, `required`, and `content`:
+
+  ```ts
+    requestBodySpec: {
+    description: 'a user',
+    required: true,
+    content: {
+      'application/json': {...schemaSpec},
+      'application/text': {...schemaSpec},
+    },
+  }
+  ```
+
+  When the type of decorated argument can be described as a schema, you can define 
```

Comment:
This sentence needs to be reworded to something like this: `When the decorated argument is a custom type or a model, make sure to decorate the model class with '@model' and '@property'`

Reply:
@shimks Hmm...I would rephrase "make sure" to "we suggest", providing a schema spec without a model class is still an option...

Reply:
"it is recommended"?  
It's a bit strange to me for using "we" in the official documentation.